libs-scala/contextualized-logging/BUILD.bazel

```diff
@@ -37,6 +37,8 @@ da_scala_test_suite(
     name = "contextualized-logging-tests",
     srcs = glob(["src/test/suite/**/*.scala"]),
     scala_deps = [
+        "@maven//:com_typesafe_akka_akka_actor",
+        "@maven//:com_typesafe_akka_akka_stream",
```

Comment:
I don't think that this library should depend on Akka. Do you think you could move the helper to the only place where it's used?

Reply:
It already depends on Akka (which is a bit strange, I agree). I've only added Akka to test suite dependencies. I could extract stream utils into a separate helper but I'm not sure if that's relevant for this PR. What do you think?

Reply:
Ha, you're right, thanks for making me notice. Feel free to split the Akka part off, but don't feel forced to. 🙂

Reply:
I'll address that in a separate PR.